aws/resource_aws_lb_listener_rule.go

```diff
@@ -154,7 +154,7 @@ func resourceAwsLbListenerRuleRead(d *schema.ResourceData, meta interface{}) err
 
 	// Rules are evaluated in priority order, from the lowest value to the highest value. The default rule has the lowest priority.
 	if *rule.Priority == "default" {
-		d.Set("priority", 99999)
+		d.Set("priority", 50000)
```

Comment:
🤔  In thinking about this a lot more, I think this will actually be a breaking change for anyone who managed to import a listener `default` rule (then set `priority = 99999` in their configuration to match). We would need to migrate their state and note the "breaking" change.

That said, I just verified its technically possible to have a `default` priority rule and a `50000` priority rule, so changing default rules to `50000` could also be confusing as its also "incorrect". 🤷‍♂️ 

Maybe the most correct answer here is changing the attribute to type string and accepting `default` priority, but only for import. Or having a separate, special `aws_lb_listener_default_rule` resource (the API does error if you try to delete default rules). This is all turning into a lot of work though 😓 .

I'm not sure how you'd want to proceed but the quick way to fix the actual validation except default rules is to leave default rules set to 99999 and go back to the custom validation function where it allows 99999 along with the _correct_ priority range of 1 through 50000. 🤕